src/app/beer_garden/api/stomp/manager.py

```diff
@@ -159,15 +160,16 @@ def _event_handler(self, event):
                     garden_name=event.payload.name, skip_key=skip_key
                 )
 
-        for value in self.conn_dict.values():
-            conn = value["conn"]
-            if conn:
-                if conn.is_connected():
-                    if value["headers_list"]:
-                        for headers in value["headers_list"]:
-                            conn.send(event, headers=headers)
-                    else:
-                        conn.send(event)
+        if not is_in_command_black_list(event):
+            for value in self.conn_dict.values():
+                conn = value["conn"]
+                if conn:
+                    if conn.is_connected():
+                        if value["headers_list"]:
+                            for headers in value["headers_list"]:
+                                conn.send(event, headers=headers)
+                        else:
```

Comment:
The first two `if`s can be collapsed into
```python
if conn and conn.is_connected():
    if value["headers_list"]:
        for header ...
    else :
        ...
```
I feel that -- in the general case -- it makes it clearer when multiple tests need to be true in order for code to run that the tests should appear together. (though this is totally stylistic preference, so it's up for debate)

On the other hand, I think it would be better if we try to avoid things like,
```python
if conn:
```
in favor of
```python
if conn is not None:
```
Also as a general rule, I think that a lot of danger for slippery bugs lies in *truthiness* checks. I feel it's always better to be explicit in what you're checking. So the same would apply to `if value["headers_list"]`.

Reply:
I think that bit only changed in the diff because it got nested under an additional if check.  It's otherwise unchanged from before, so probably should stay that way.